README.md

```diff
@@ -1 +1,14 @@
-# DeepMEM
\ No newline at end of file
+# DeepMEM
+
+[![GitHub Project](https://img.shields.io/badge/GitHub--blue?style=social&logo=GitHub)](https://github.com/mihirkatare/DeepMEM)
```

Comment:
There's nothing wrong with this badge, but as it isn't clear if this project will get deployed to PyPI or someplace where the README will get duplicated it isn't really necessary to have a project badge linking back here as it is just self referential at this point.